KYHBPA_TeamA/KYHBPA_TeamA/Controllers/PhotoController.cs

```diff
@@ -172,7 +173,8 @@ public ActionResult Create(AddPhotoViewModel addViewModel, HttpPostedFileBase im
                     PhotoDesc = addViewModel.Description,
                     PhotoData = new byte[image.ContentLength],
                     PhotoTitle = addViewModel.Title,
-                    MimeType = image.ContentType
+                    MimeType = image.ContentType,
```

Comment:
I think that's so the file type is stored. I think Pedro added it, so that only pictures were uploaded to the image gallery. Not really sure what your question is asking.

Reply:
It's the end of a list shouldn't the comma be removed

Reply:
You're right. I'll take it off.